example/tutorial/05_simple_atomics/simple_atomics.cpp

```diff
@@ -48,17 +48,17 @@
 #include <cmath>
 
 // Type of a one-dimensional length-N array of int.
-typedef Kokkos::View<int*> view_type;
-typedef view_type::HostMirror host_view_type;
+using view_type      = Kokkos::View<int*>;
+using host_view_type = view_type::HostMirror;
 // This is a "zero-dimensional" View, that is, a View of a single
 // value (an int, in this case).  Access the value using operator()
 // with no arguments: e.g., 'count()'.
 //
 // Zero-dimensional Views are useful for reduction results that stay
 // resident in device memory, as well as for irregularly updated
 // shared state.  We use it for the latter in this example.
-typedef Kokkos::View<int> count_type;
-typedef count_type::HostMirror host_count_type;
+using count_type      = Kokkos::View<int>;
+using host_count_type = count_type::HostMirror;
```

Comment:
I'm kind of surprised this doesn't require `typename`?

Reply:
Same here.